crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -15,6 +15,8 @@
 #
 
 class Api::UpgradeController < ApplicationController
+  before_action :set_content_type
```

Comment:
Maybe it's a proper time to abstract this thing away in `ApiController < ApplicationController` somehow, and subclass from that `ApiContoller` everywhere?

This `before_action` everywhere seems to be very easy to forget at some point...